Add web sockets support (#128)
python-lsp/python-lsp-server: README.md

- [autopep8](https://github.com/hhatto/autopep8) for code formatting
- [YAPF](https://github.com/google/yapf) for code formatting (preferred over autopep8)

> PYLSP can now be used in Web Sockets based configuration as follows:

Reviewer: I think we do not use PYLSP in capitals elsewhere so maybe use the full name?

Reviewer: ```suggestion
Python LSP Server can communicate over WebSockets when configured as follows:
```

```
pylsp --ws --host [host/ip] --port [port]
```

The following libraries are required for Web Sockets support:
- [autobahn-twisted](https://github.com/crossbario/autobahn-python) for PYLSP Web sockets using autobahn in twisted configuration. refer [Autobahn installation](https://autobahn.readthedocs.io/en/latest/installation.html) for more details

you can install these dependencies with below command:

> 
> ```
> pip install autobahn[twisted,accelerate]>=21.11.1
> ```
> 
> or with this:

Reviewer: Let's keep it simple and easier to maintain.

```suggestion
```

```
pip install 'python-lsp-server[autobahn-twisted]'
```

> 

Reviewer: Let's move this block to the end of `Configuration` section -  an average user does not care what method is used.

3. Same as 1, but add to `setup.cfg` file in the root of the project.

Python LSP Server can communicate over WebSockets when configured as follows:

```
pylsp --ws --host [host/ip] --port [port]
```

The following libraries are required for Web Sockets support:

> - [autobahn-python](https://github.com/crossbario/autobahn-python) for Python LSP Server Web sockets using autobahn with twisted configuration. refer [Autobahn installation](https://autobahn.readthedocs.io/en/latest/installation.html) for more details

Reviewer 2: ```suggestion
The [autobahn-python](https://github.com/crossbario/autobahn-python) library is required for Web Sockets support. Refer to the [Autobahn installation](https://autobahn.readthedocs.io/en/latest/installation.html) for more details.
```

> you can install these dependencies with below command:

Reviewer 2: ```suggestion
You can install this dependency with command below:
```

Reviewer 2: Please address this suggestion.
